src/perl/vdotest/VDOTest/VDOFormatInKernel.pm

```diff
@@ -0,0 +1,176 @@
+##
+# Test the vdoFormat application
```

Comment:
The erl test might also write some junk to the device beforehand, to make sure we're correctly formatting the stuff we need to.

Reply:
We don't tests that do that for vdoformat either for that matter.